Dockerfile

```diff
@@ -1,8 +1,12 @@
-FROM checksum/python-newspaper:latest
+FROM python:3.7.0-stretch
```

Comment:
- could you check in `-slim-stretch` works because it’s lighter in terms of size?
- also lets use `3.7-stretch` to be able to automatically get the patches on rebuild

Reply:
This is what I get with slim
Exception: you need a C compiler to build uWSGI

stretch just works.

Reply:
Removed the patch version